integreat_cms/static/src/css/style.scss

```diff
@@ -573,14 +573,14 @@ datalist option {
 }
 
 /* Special word break for feedback list */
```

Comment:
Maybe this comment could be updated since this feature is now used for more than just the feedback list?